src/meta/processors/admin/CreateSnapshotProcessor.cpp

```diff
@@ -31,7 +31,7 @@ void CreateSnapshotProcessor::process(const cpp2::CreateSnapshotReq&) {
   }
 
   auto snapshot = folly::sformat("SNAPSHOT_{}", MetaKeyUtils::genTimestampStr());
-  folly::SharedMutex::WriteHolder wHolder(LockUtils::snapshotLock());
+  folly::SharedMutex::WriteHolder holder(LockUtils::lock());
```

Comment:
Ask a question:
create snapshot and create backup is a time-consuming operation. This lock will be held here for a long time.
During this period, other heartbeats may fail?

Reply:
Good point. I don't know the time-consuming in the case of a large amount of data.